app/assets/javascripts/settings.coffee

```diff
@@ -11,9 +13,26 @@ parseFloatOrElse = (str, def) ->
 clamp = (min, max, val) ->
   Math.max(min, Math.min(max, val))
 
+updatedLocales = locales.map (locale) ->
```

Comment:
So this is closer to what I meant, but I actually meant just hard-coding the `languageCode` values on the objects themselves in the `settings-controller.coffee` file.  So no CoffeeScript code should be required to do it programmatically.